### PR TITLE
Wi 9

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: modules/data/src/main/java/io/fluo/webindex/data/spark/IndexUtil.java [hunk 1]
```diff
@@ -145,28 +132,76 @@ private static void addRCV(List<Tuple2<RowColumn, Long>> tuples, String r, Colum
...
+            .filter(t -> !t._1().getColumn().getFamily().toString().equals(Constants.RANK));
+    return fluoIndex;
+  }
+
+  static class RowColumnComparator implements Comparator<RowColumn>, Serializable {
+    final static RowColumnComparator INSTANCE = new RowColumnComparator();
+
+    @Override
+    public int compare(RowColumn o1, RowColumn o2) {
+      int result;
```
Comment (keith-turner) [inline modules/data/src/main/java/io/fluo/webindex/data/spark/IndexUtil.java]: could this return `o1.compareTo(o2)` ?

File: modules/data/src/test/java/io/fluo/webindex/data/spark/IndexUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  public void tearDown() {
+    sc.stop();
+    sc = null;
+  }
+
+  @Test
+  public void testDataSet1() throws Exception {
+    JavaRDD<Page> pages = sc.parallelize(getPagesSet1());
+    IndexStats stats = new IndexStats(sc);
+    JavaPairRDD<RowColumn, Bytes> accumuloIndex = IndexUtil.createAccumuloIndex(stats, pages);
```
Comment (keith-turner) [inline modules/data/src/test/java/io/fluo/webindex/data/spark/IndexUtilTest.java]: I have no idea what these three lines are doing :)  Would you mind adding some javadoc explaining the flow?

Comment (keith-turner) [inline modules/data/src/test/java/io/fluo/webindex/data/spark/IndexUtilTest.java]: Well three pairs of lines

**Comment by keith-turner** [comment]
+1

File: modules/data/src/main/java/io/fluo/webindex/data/spark/IndexUtil.java (hunk 1 above)
Comment (mikewalch) [inline modules/data/src/main/java/io/fluo/webindex/data/spark/IndexUtil.java]: Good catch.  The custom comparator is no longer needed now that I made RowColumn comparable in Fluo.  I removed the comparator in de9fe46c12dcc8c0c6

File: modules/data/src/test/java/io/fluo/webindex/data/spark/IndexUtilTest.java (hunk 2 above)
Comment (mikewalch) [inline modules/data/src/test/java/io/fluo/webindex/data/spark/IndexUtilTest.java]: Improved documentation in de9fe46c12dcc8c0c6

**Pull request merged**